### ISSUE TITLE
Change name of best-knowledge madx file.

### ISSUE THREAD

**Issue by JoschD** [pylhc/omc3#331]
job.create_model.madx is overwritten by best-knowledge model creator (the two creators are called by the GUI in a row). 
Maybe should be job.create_bestknowledge_model.madx (see other model creators in python code on how to easily overwrite)

**Comment by awegsche** [comment]
yes, `beta_from_phase` for example uses the best knowledge model for its model phase advances:

https://github.com/pylhc/omc3/blob/6763e865ce40ea0c34d02d1c5039574004c677ef/omc3/optics_measurements/beta_from_phase.py#L489-L493

which is ultimately used in lines 84 und 85 of
https://github.com/pylhc/omc3/blob/6763e865ce40ea0c34d02d1c5039574004c677ef/omc3/optics_measurements/beta_from_phase.py#L69

**Comment by awegsche** [comment]
In general it should be used by every model-dependent algorithm like `beta_from_phase` and both AC-dipole compensation methods (especially Ryoichi's)
